### DISCUSSION TITLE
MLS Method

### DISCUSSION THREAD

**Discussion by gzalles** [thomasdeppisch/hoast360#15] [General]
Hi, it's me again. I was wondering if there is a public repo that shows how to perform the MLS method. I read the paper but noticed there is no codebase. 

Just wondering.

Thanks a million, Gabe

**Comment by thomasdeppisch** [comment]
Hi Gabe,

Unfortunately there is no implementation available from the authors.
I re-implemented magLS as described in the [Ambisonics book](https://www.springer.com/gp/book/9783030172060) in 4.11.2. If you drop me an email I can send you my implementation but at the moment I do not want to share it publicly (actually I am planning to publish a modified version of it soon).

Thomas

**Reply by gzalles** [reply]
Hey Thomas, 

Sorry for the late reply. I would love to receive your code if you could. The reference definitely helps but I am not 100% I understood the equation. 

Good luck with your research, excited to read it when it comes out.

My email is gzalles at ucsd dot edu. 🎹
